components/AppBar/README.md

```diff
@@ -153,90 +124,496 @@ import MaterialComponents.MaterialAppBar
 
 ## Usage
 
-<!-- Extracted from docs/typical-use-adding-an-app-bar.md -->
+<!-- Extracted from docs/typical-use-navigation-controller.md -->
 
-### Typical use: Adding an app bar to your app
+### Typical use: View controller containment, as a navigation controller
 
-Each view controller in your app that intends to use an App Bar will follow these instructions.
-You'll typically add the App Bar to the same view controllers that you'd push onto a
-UINavigationController, hiding the UINavigationController's `navigationBar` accordingly.
+The easiest integration path for using the app bar is through the `MDCAppBarNavigationController`.
+This API is a subclass of UINavigationController that automatically adds an
+`MDCAppBarViewController` instance to each view controller that is pushed onto it, unless an app bar
+or flexible header already exists.
 
-The result of following these steps will be that:
+When using the `MDCAppBarNavigationController` you will, at a minimum, need to configure the added
+app bar's background color using the delegate.
 
-1. an App Bar is registered as a child view controller of your view controller,
-2. you have access to the App Bar's Flexible Header view via the headerViewController property, and
-   that
-3. you have access to the Navigation Bar and Header Stack View views via the corresponding
-   properties.
+#### Example
 
-Step 1: **Create an instance of MDCAppBar**.
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+let navigationController = MDCAppBarNavigationController()
+navigationController.pushViewController(<#T##viewController: UIViewController##UIViewController#>, animated: <#T##Bool#>)
```

Comment:
Is it supposed to have all the "#"?

Also, in Obj-C example

Reply:
Yep these are code auto-completion markers. If you copy-paste this snippet into Xcode it'll allow you to tab-complete each of the markers.